manifest.wcf.json

```diff
@@ -0,0 +1,193 @@
+{
+  "readmePath": "README.md",
+  "registry": "mcr.microsoft.com",
+  "variables": {
+    "WcfReleaseDateStamp": "20190709"
+  },
+  "repos": [
+    {
+      "id": "wcf",
+      "name": "dotnet/framework/wcf",
+      "readmePath": "README.wcf.md",
+      "mcrTagsMetadataTemplatePath": "eng/mcr-tags-metadata-templates/wcf-tags.yml",
+      "images": [
+        {
+          "sharedTags": {
+            "4.8": {},
+            "latest": {}
+          },
+          "platforms": [
+            {
+              "dockerfile": "4.8/wcf/windowsservercore-1803",
+              "os": "windows",
+              "osVersion": "windowsservercore-1803",
+              "tags": {
+                "4.8-$(WcfReleaseDateStamp)-windowsservercore-1803": {},
+                "4.8-windowsservercore-1803": {}
+              }
+            },
+            {
+              "dockerfile": "4.8/wcf/windowsservercore-ltsc2016",
+              "os": "windows",
+              "osVersion": "windowsservercore-ltsc2016",
+              "tags": {
+                "4.8-$(WcfReleaseDateStamp)-windowsservercore-ltsc2016": {},
+                "4.8-windowsservercore-ltsc2016": {}
+              }
+            },
+            {
+              "dockerfile": "4.8/wcf/windowsservercore-ltsc2019",
+              "os": "windows",
+              "osVersion": "windowsservercore-ltsc2019",
+              "tags": {
+                "4.8-$(WcfReleaseDateStamp)-windowsservercore-ltsc2019": {},
+                "4.8-windowsservercore-ltsc2019": {}
+              }
+            },
+            {
+              "dockerfile": "4.8/wcf/windowsservercore-1903",
+              "os": "windows",
+              "osVersion": "windowsservercore-1903",
+              "tags": {
+                "4.8-$(WcfReleaseDateStamp)-windowsservercore-1903": {},
+                "4.8-windowsservercore-1903": {}
+              }
+            }
+          ]
+        },
+        {
+          "sharedTags": {
+            "4.7.2": {}
+          },
+          "platforms": [
+            {
+              "dockerfile": "4.7.2/wcf/windowsservercore-ltsc2016",
+              "os": "windows",
+              "osVersion": "windowsservercore-ltsc2016",
+              "tags": {
+                "4.7.2-$(WcfReleaseDateStamp)-windowsservercore-ltsc2016": {},
+                "4.7.2-windowsservercore-ltsc2016": {}
+              },
+              "customBuildLegGrouping": [
```

Comment:
I suspect this isn't needed.  What this does is allow the public PR build to build additional images that this image doesn't have a dependency on but are necessary to be built in order to test.  This is done in our base .NET Fx images because we have scenarios that test apps on runtime versions where the apps were built using a different SDK version.  So I would suggest remove all these `customBuildLegGrouping` in the manifest.  And also change the pipeline files to not indicate that a `customBuildLegGrouping` should be used for the build.  If you create a public pipeline, you can actually verify that this runs correctly.

Reply:
Fixed in #330